<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-9BB5NPN" failures="0" tests="7" name="project_life_insurance.AddPaymentUsingAnnotationTest" time="10.015" errors="1" timestamp="2023-01-16T17:33:18 IST" skipped="6">
  <testcase classname="project_life_insurance.AddPaymentUsingAnnotationTest" name="simple4" time="0.000">
    <skipped/>
  </testcase> <!-- simple4 -->
  <system-out>
    <![CDATA[--TestScript Execution skipped--
simple5-----> Testscript execution started]]>
  </system-out>
  <testcase classname="project_life_insurance.AddPaymentUsingAnnotationTest" name="simple3" time="0.000">
    <skipped/>
  </testcase> <!-- simple3 -->
  <system-out>
    <![CDATA[--TestScript Execution skipped--
simple4-----> Testscript execution started]]>
  </system-out>
  <testcase classname="project_life_insurance.AddPaymentUsingAnnotationTest" name="addPayment" time="0.000">
    <skipped/>
  </testcase> <!-- addPayment -->
  <system-out>
    <![CDATA[--TestScript Execution skipped--
simple-----> Testscript execution started]]>
  </system-out>
  <testcase classname="project_life_insurance.AddPaymentUsingAnnotationTest" name="simple2" time="0.000">
    <skipped/>
  </testcase> <!-- simple2 -->
  <system-out>
    <![CDATA[--TestScript Execution skipped--
simple3-----> Testscript execution started]]>
  </system-out>
  <testcase classname="project_life_insurance.AddPaymentUsingAnnotationTest" name="simple" time="0.000">
    <skipped/>
  </testcase> <!-- simple -->
  <system-out>
    <![CDATA[--TestScript Execution skipped--
simple2-----> Testscript execution started]]>
  </system-out>
  <testcase classname="project_life_insurance.AddPaymentUsingAnnotationTest" name="simple5" time="0.000">
    <skipped/>
  </testcase> <!-- simple5 -->
  <system-out>
    <![CDATA[--TestScript Execution skipped--]]>
  </system-out>
  <testcase classname="project_life_insurance.AddPaymentUsingAnnotationTest" name="configBC" time="10.015">
    <error message="unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=108.0.5359.126)
Build info: version: &#039;4.4.0&#039;, revision: &#039;e5c75ed026a&#039;
System info: host: &#039;DESKTOP-9BB5NPN&#039;, ip: &#039;192.168.0.191&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.3&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [1d0ad2c650e7f453da10efaa833a44ab, get {url=http://rmgtestingserver/domain/Life_Insurance_Management_System/}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 108.0.5359.126, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: C:\Users\hp\AppData\Local\T...}, goog:chromeOptions: {debuggerAddress: localhost:65317}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:65317/devtoo..., se:cdpVersion: 108.0.5359.126, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 1d0ad2c650e7f453da10efaa833a44ab" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=108.0.5359.126)
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'DESKTOP-9BB5NPN', ip: '192.168.0.191', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.3'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [1d0ad2c650e7f453da10efaa833a44ab, get {url=http://rmgtestingserver/domain/Life_Insurance_Management_System/}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 108.0.5359.126, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: C:\Users\hp\AppData\Local\T...}, goog:chromeOptions: {debuggerAddress: localhost:65317}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:65317/devtoo..., se:cdpVersion: 108.0.5359.126, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 1d0ad2c650e7f453da10efaa833a44ab
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:310)
at com.insurance.GenericUtilities.BaseClass.configBC(BaseClass.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- configBC -->
  <system-out/>
</testsuite> <!-- project_life_insurance.AddPaymentUsingAnnotationTest -->
